python/README.rst

```diff
@@ -103,10 +103,33 @@ For Mac OSX
     # or for python3 (you may have to uninstall boost and reinstall to build python3 libs)
     $ brew install boost-python3
 
-Also, having Anaconda in your path can cause segmentation faults when importing the pyvw module. Providing Conda support
-is an open issue and efforts are welcome, but in the meantime it is suggested to remove any conda bin directory from your path
-prior to installing the vowpalwabbit package.
+Having `conda` on a Linux-based system requires a build with conda-provided packages to ensure consistency:
 
+.. code-block:: bash
+
+    # create/activate a conda environment (depends on your choice, no example is given)
+    # boost packages
+    conda install boost libboost py-boost
+    # a consistent compiler (g++ 7.2.0)
+    conda install gxx_linux-64
+    # pick up the newly installed compiler
+    source activate YourEnvironmentName
+    # make a soft link to the compiler, since Makefiles internally use `which g++`
+    [ ! -z ${GXX} ] && ln -s ${GXX} `dirname ${GXX}`/g++
+    # set BOOST_XXX variables, that will be used to find boost libs in linking
+    export BOOST_INCLUDE='-I /your_conda_env_path/include/boost -I /your_conda_env_path/include'
+    export BOOST_LIBRARY='-L /your_conda_env_path/lib'
+    # get the package from github
+    git clone https://github.com/JohnLangford/vowpal_wabbit.git
+    # build the C++ binaries and python package
+    cd vowpal_wabbit
+    python python python/setup.py install
+
+This procedure was tested with python3, but is expected to also work with python2.
+Typically, you can find `/your_conda_env_path` by either checking the `$CONDA_PREFIX` environmental variable
+or `conda info --base` if you do not have any environments (beware, environments make your life better).
+
+ 
```

Comment:
@arielf There is no conda script provided, but there are instructions added to the docs. I can complete those instructions to automatically deduce `BOOST_XXX from the python path as you have suggested. Did you suggest a real script in the repository? I do not know what are the policies about it and if that is encouraged/allowed?

Reply:
Thanks @mlisovyi 
Ideally the build is uniform/short and can be supported directly in the `Makefile` with only some env var changes.
But since anaconda is so involved, have it own tools and need additional operations, it may be preferable to keep all this unusual and non-standard complexity out of the standard `Makefiles`.  The procedure itself, may call some make targets (if they work as is, there's no need to repeat them in the conda script) but given the unique differences, I'd personally prefer a separate `conda-install.sh` over filling the Makefiles with env-specific cruft.

Reply:
@arielf could you please suggest where would be an appropriate location to place such a script? I would imagine directly in the `python` folder, as conda is used by python users.

Reply:
I think both in the python dir or the top dir would make sense. top dir may be more visible/obvious to new anaconda users trying to figure out things.  Please make sure to name the script clearly (like the suggestion above, or similar) so its purpose is obvious for random observers. Thanks for all your help on this. Anaconda has been a pain point for many, but no one of the problem reporters, stakeholders, has contributed a solution. Really glad someone is taking responsibility and fixing this  :+1: